src/scripts/main.js

```diff
@@ -1 +1,20 @@
 'use strict';
+
+document.addEventListener('DOMContentLoaded', () => {
+  const largeImg = document.getElementById('largeImg');
+  const thumbs = document.getElementById('thumbs');
+
+  // eslint-disable-next-line no-shadow
+  thumbs.addEventListener('click', (event) => {
+    event.preventDefault();
```

Comment:
The `event.preventDefault();` is used here to prevent the default action of the anchor tag. This is correct if the intention is to prevent navigation when a thumbnail is clicked. Ensure this behavior is intended, as it will stop the browser from following the link.